tox/venv.py

```diff
@@ -289,6 +290,14 @@ def run_install_command(self, packages, action, options=()):
             i = argv.index('{opts}')
             argv[i:i + 1] = list(options)
 
+        if '{constraints}' in argv:
```

Comment:
If I see it correct, this is the actual feature improvement implementation part. 
It seems to be correct, that it adds ``-c constraints`` to the install_command list.

But I did not see anything about the described cases, that the tested package itself is listed in the constraints file. Could you please correct me, but I did not see anything that makes a different to add a file to the constraints parameter than adding -cconstraints.txt to deps parameter. 

It might be more declarative and explicit that before, but I did not see the described case handled.

Reply:
I think you're correct.  Adding -c to the deps does seem to do handle the situation I'm trying to deal with.

Should I remove this PR while I investigate?